src/Http/Controllers/MediaController.php

```diff
@@ -24,6 +26,10 @@ public function create(
         $file = $request->file('file');
         $data = $request->onlyValidated();
 
+        if (!empty($data['preview_drivers'])) {
```

Comment:
as discussed, previews types should don be passed via the request, only as argumets of create and bulkCreate methods